docs/modules/releasenotes/pages/changelog.adoc

```diff
@@ -2,6 +2,46 @@
 [[release-31-changelog]]
 = Changelog
 
+[[releasenotes-changelog-31.0.5]]
+
+==== Release 31.0.5
```

Comment:
the number of `=` is not correct (expecting to see `==` instead of `===`).

Reply:
Pushed a fix, thanks.